ignite/pkg/cosmosclient/cosmosclient_test.go

```diff
@@ -0,0 +1,369 @@
+package cosmosclient
```

Comment:
```suggestion
package cosmosclient_test
```

Reply:
Good point, I agree on making explicit when tests are only testing the "public package API", but in this case I would have to make the `createTxSearchByHeightQuery()` public, which I prefer to keep internal to the package. I changes the TX one to be explicitly public.